packages/core/src/components/templates/PeriodInput/PeriodInput.tsx

```diff
@@ -23,6 +23,8 @@ import {DateInput} from '../../organisms';
 import {getEndOfDay, getStartOfDay} from '../../../utils';
 import {useTranslator} from '../../../i18n';
 
+const hoursToMilliseconds = (hours: number) => hours * 60 * 60 * 1000;
```

Comment:
Maybe we can move this to the time utils file in root utils folder as it is not linked to the component